### PR TITLE
Try to make script work on ec2 and virtualbox vms

### PR DESCRIPTION
**Pull request by athompson-r7**
This commit aims to make the shell script to set permission for artifact
copying work on both ec2 and virtualbox vms.

### CODE REVIEW

**Comment by rvanburen-r7** [comment]
:+1:

**Comment by jmanero-r7** [comment]
I think the method used in the packer interface should work here as well: https://github.com/rapid7/builderator/blob/master/lib/builderator/interface/packer.rb#L35-L36

File: template/Vagrantfile.erb [hunk 1]
```diff
@@ -37,11 +37,23 @@ Vagrant.configure('2') do |config|
 
   # config.vm.network :forwarded_port, :host => 9200, :guest => 9200
 
+  $script = %q{
```
Comment (erran-r7) [inline template/Vagrantfile.erb]: Why not create a constant instead of a global variable?

**Comment by athompson-r7** [comment]
I like that.  We need rather open perms because we scp the bundle to instance.  I might combine the two.

File: template/Vagrantfile.erb (hunk 1 above)
Comment (athompson-r7) [inline template/Vagrantfile.erb]: I was simply following an example on the vagrant website :/  If folks feel strongly I will change it.

**Comment by jmanero-r7** [comment]
Setting the owner to the user that Vagrant is provisioning with should handle all of the other permission issues. `mkdir` defaults to `0755` AFAIK?

**Comment by jmanero-r7** [comment]
@athompson-r7, Any objection to merging this guy?

**Pull request merged**